imports/plugins/core/orders/server/no-meteor/resolvers/OrderItem/productTags.js

```diff
@@ -0,0 +1,21 @@
+import { getPaginatedResponse } from "@reactioncommerce/reaction-graphql-utils";
+import { xformArrayToConnection } from "@reactioncommerce/reaction-graphql-xforms/connection";
+
+/**
+ * @name "OrderItem.productTags"
+ * @method
+ * @memberof Catalog/GraphQL
+ * @summary Returns the tags for an OrderItem
+ * @param {Object} order - OrderItem from parent resolver
+ * @param {TagConnectionArgs} args - arguments sent by the client {@link ConnectionArgs|See default connection arguments}
+ * @param {Object} context - an object containing the per-request state
+ * @return {Promise<Object[]>} Promise that resolves with array of Tag objects
+ */
+export default async function tags(OrderItem, connectionArgs, context) {
```

Comment:
Change the function name to `productTags`

Reply:
Also, `orderItem` should be camelCase and doesn't match the `@param` name in the jsdoc